crates/bevy_internal/src/default_plugins.rs

```diff
@@ -13,6 +13,9 @@ impl PluginGroup for DefaultPlugins {
         group.add(bevy_asset::AssetPlugin::default());
         group.add(bevy_scene::ScenePlugin::default());
 
+        #[cfg(feature = "bevy_animation_rig")]
```

Comment:
You'll need to describe this new feature in https://github.com/bevyengine/bevy/blob/main/docs/cargo_features.md